src/workflows-app/SubmissionHistory.js

```diff
@@ -231,15 +235,23 @@ export const BaseSubmissionHistory = ({ namespace, workspace }, _ref) => {
                         },
                         ['No workflows match the given filter.']
                       )
-                    : div(['See workflows that were submitted by all collaborators in this workspace.']),
+                    : div({}, [
+                        div(['See workflows that were submitted by all collaborators in this workspace.']),
+                        div(['Your workspace nickname is ', span({ style: { fontWeight: 'bold' } }, [samIdToAnonymousName(userId)]), '.']),
```

Comment:
Are these nickname same for a user across different workspaces? If yes, would it make sense to call describe it as `Your nickname is..`? The word workspace makes it seem a user will have different nickname per workspace.

Reply:
I see what you're saying. When I wrote "workspace nickname" I was thinking something like: "this nickname represents you inside the workspace"... maybe we can get Sara or UX's opinion

(there's also a question of whether or not these nicknames *should* be the same across workspaces... I think it's a good thing, and a user said they want it that way, but I'm curious for other opinions)

Reply:
having thought about it a little more: I think it's safer to keep the word "workspace" in the phrase "workspace nickname", in case there is a need to make these nicknames shift across workspaces. With this wording, we don't guarantee that they remain the same across workspaces. The fact that they are the same doesn't invalidate the fact that these are "workspace nicknames", in my opinion

Reply:
I interpreted "workspace nickname" as my nickname in **this** workspace which makes it sound I will have different nicknames in different workspaces. If nickname is same for user across workspaces maybe it can be "Your nickname is..". But definitely worth running it by Sara and UX.

> there's also a question of whether or not these nicknames should be the same across workspaces

My opinion is that user should have the same nickname across all workspaces to avoid confusion and keep it simple. But I will defer that to Sara and UX as well.